### ISSUE TITLE
Code for "Advancing Referring Expression Segmentation Beyond Single Images"

### ISSUE THREAD

**Issue by RoadoneP** [shikras/d-cube#14]
Thank you for releasing your dataset. I can't find the model proposed in the paper "Advancing reference expression segmentation beyond single images" in your code. If possible, could you please provide the code for the model and train.py?

**Comment by xiao-11** [comment]
Similarly, I couldn't find it either about the code of "Advancing reference expression segmentation beyond single images".

**Comment by Charles-Xie** [comment]
Please see the previous answer (https://github.com/shikras/d-cube/issues/6#issuecomment-1809662054) from Yixuan Wu in issue 6. You can send email to [Yixuan Wu](mailto:2948819544@qq.com) for the code of "Advancing Referring Expression Segmentation Beyond Single Images".

**Comment by RoadoneP** [comment]
Yes, but I have sent the email twice and still haven't received a reply.
